integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/GroupResource.java

```diff
@@ -90,12 +90,12 @@ public interface GroupResource {
      *
      * @param first
      * @param max
-     * @param full
+     * @param briefRepresentation
```

Comment:
Might as well remove these comments since they don't add any additional information.

Reply:
Thanks! I consider it possibly as a follow-up if we want to review javadoc of the admin-client in general as there are lots of the methods, which use empty parameters. I've just changed this to remove warning from Intellij Idea due to incorrect parameter name.